homeassistant/components/samsungtv/bridge.py

```diff
@@ -123,6 +123,8 @@ def is_on(self):
 
     def send_key(self, key):
         """Send a key to the tv and handles exceptions."""
+        if self._remote is None:
```

Comment:
Be aware there there are 2 send_key functions: one for Legacy TV and one for WebSocket Tv.

Simone